integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
Those should show the entire output stream - including all of the build steps. Can you show me the exact code you are using when you call buildImageWithOut?  Did you try the code I pasted?